testutils/clustermanager/gke.go

```diff
@@ -89,6 +91,14 @@ func (gs *GKEClient) Setup(r GKERequest) ClusterOperations {
 		gc.NeedsCleanup = true
 	}
 
+	if r.ClusterName == "" {
```

Comment:
can't comment there, but why is line 75 pointer?

Reply:
I think just checking "" should be enough, since empty is not a valid value anyway, as far as I can tell?

Reply:
So, we really don't want this one to be fixed?

Reply:
Overlooked, done